lms/djangoapps/teams/tests/test_views.py

```diff
@@ -669,7 +669,7 @@ def build_team_data(
             'name': name,
             'course_id': str(course.id),
             'description': description,
-            'topic_id': topic_id,
+            'teamset_id': teamset_id,
```

Comment:
I think this may be causing some errors. I'd just leave this as is for now

Reply:
Reverted back to topic_id